LoRaEngine/LoraKeysManagerFacade/FacadeStartup.cs

```diff
@@ -25,12 +31,17 @@ public override void Configure(IFunctionsHostBuilder builder)
             if (builder is null) throw new ArgumentNullException(nameof(builder));
 
             var configHandler = ConfigHandler.Create(builder);
-
             var iotHubConnectionString = configHandler.IoTHubConnectionString;
-            if (iotHubConnectionString == null)
+            var iotCentralEndpoint = configHandler.IoTCentralEndpoint;
+
+            if (configHandler.DeviceRegistryMode == DeviceRegistryMode.IoTHub && iotHubConnectionString == null)
             {
                 throw new InvalidOperationException($"Missing {ConfigHandler.IoTHubConnectionStringKey} in settings");
             }
+            else if (configHandler.DeviceRegistryMode == DeviceRegistryMode.IoTCentral && iotCentralEndpoint == null)
```

Comment:
related to a previous conversation as IoT Central is using DPS, what is IoT Central related that cannot be just DPS?